binding-library/java/src/main/java/com/microsoft/azure/functions/kafka/KafkaEventData.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.kafka;
+
+/**
+ * KafkaOutput Event Data to use with KafkaOutput binding
+ */
+public class KafkaEventData {
+    /**
+     * Constructor
```

Comment:
Do not add KafkaEventData . The right approach is follow similar pattern as C# and use Generics and not object

Reply:
You right. I remove KafkaEventData. We can use byte[] or String for it. Thank you for the review. Today I don't have time during the daytime. I'll do it after coming back to my house.